pybamm/models/submodels/interface/sei/sei_growth.py

```diff
@@ -113,33 +141,47 @@ def get_coupled_variables(self, variables):
             # so
             #  j_sei = -C_sei_ec * exp() / (1 + L_sei * C_ec * C_sei_ec * exp())
             #  c_ec = 1 / (1 + L_sei * C_ec * C_sei_ec * exp())
-            # need to revise for thermal case
-            C_sei_exp = C_sei_ec * pybamm.exp(-0.5 * (delta_phi - j * L_sei * R_sei))
+            C_sei_exp = C_sei_ec * pybamm.exp(-0.5 * prefactor * eta_SEI)
             j_sei = -C_sei_exp / (1 + L_sei * C_ec * C_sei_exp)
             c_ec = 1 / (1 + L_sei * C_ec * C_sei_exp)
 
             # Get variables related to the concentration
             c_ec_av = pybamm.x_average(c_ec)
-            c_ec_scale = self.param.c_ec_0_dim
-
-            variables.update(
-                {
-                    "EC surface concentration": c_ec,
-                    "EC surface concentration [mol.m-3]": c_ec * c_ec_scale,
-                    "X-averaged EC surface concentration": c_ec_av,
-                    "X-averaged EC surface concentration [mol.m-3]": c_ec_av
-                    * c_ec_scale,
-                }
-            )
+            c_ec_scale = param.c_ec_0_dim
+
+            if self.reaction == "SEI on cracks":
+                variables.update(
+                    {
+                        "EC concentration on cracks": c_ec,
+                        "EC concentration on cracks [mol.m-3]": c_ec * c_ec_scale,
+                        "X-averaged EC concentration on cracks": c_ec_av,
+                        "X-averaged EC concentration on cracks [mol.m-3]": c_ec_av
+                        * c_ec_scale,
+                    }
+                )
+            else:
+                variables.update(
+                    {
+                        "EC surface concentration": c_ec,
+                        "EC surface concentration [mol.m-3]": c_ec * c_ec_scale,
+                        "X-averaged EC surface concentration": c_ec_av,
+                        "X-averaged EC surface concentration [mol.m-3]": c_ec_av
+                        * c_ec_scale,
+                    }
+                )
 
         if self.options["SEI"] == "ec reaction limited":
             alpha = 0
         else:
-            alpha = 0.5
+            alpha = param.alpha_sei
+
+        # All SEI growth mechanisms assumed to have Arrhenius dependence
+        Arrhenius = pybamm.exp(param.E_over_RT_sei * (1 - prefactor))
 
-        j_inner = alpha * j_sei
-        j_outer = (1 - alpha) * j_sei
+        j_inner = alpha * Arrhenius * j_sei
+        j_outer = (1 - alpha) * Arrhenius * j_sei
```

Comment:
I understand why you've done it this way, but we should reformat the SEI parameters so that the parameter is `j_sei(c, T)` to be consistent with intercalation kinetics and plating. The SEI parameters are a mess anyway.

Reply:
This is a good idea but probably beyond the scope of this PR, which is to allow others to reproduce the results of the O'Kane 2022 paper.